main.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"archive/zip"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/input"
+)
+
+const (
+	targetExtension string = ".zip"
+)
+
+// ConfigsModel ...
+type ConfigsModel struct {
+	SourcePath string
+	TargetPath string
+}
+
+func main() {
+	configs := createConfigsModelFromEnvs()
+
+	fmt.Println()
+	configs.print()
+
+	if err := configs.validate(); err != nil {
+		fail("Issue with input: %s", err)
+	}
+
+	_, err := compress(configs.SourcePath, configs.TargetPath)
+
+	if err != nil {
+		fail("Issue with compress: %s", err)
+		return
+	}
+}
+
+// Adds the file extension to the end of the targetpath if it's needed.
```

Comment:
func description starts with the function's name, like:

`fixTargetExtension adds the file extension to the end of the targetpath if it's needed.`